### ISSUE TITLE
Crash at the end of OptimizeTests.embedded_python_test

### ISSUE THREAD

**Issue by akenmorris** [SCIInstitute/ShapeWorks#1650]
On Linux in Debug mode, I'm getting:

```
    free(): invalid pointer
```

At the end of `OptimizeTests.embedded_python_test`

backtrace:

```
        at /usr/include/c++/7/ext/new_allocator.h:140
        at /usr/include/c++/7/bits/alloc_traits.h:487
        at /usr/include/c++/7/bits/forward_list.tcc:90
        at /usr/include/c++/7/bits/forward_list.h:329
        at /usr/include/c++/7/bits/forward_list.h:559
        __in_chrg=<optimized out>)
        at /home/csafe/amorris/anaconda3/envs/shapeworks/include/pybind11/detail/internals.h:113
        at /home/csafe/amorris/anaconda3/envs/shapeworks/include/pybind11/embed.h:164
        at ../Libs/Optimize/Optimize.cpp:229
```

**Comment by akenmorris** [comment]
This is apparently due to, or at least related to, mixing ITK versions in the same namespace.

The problem was fixed (14147bd) by removing an unused `import itk` in `data.py`

**Comment by akenmorris** [comment]
Fixed in #1651
